.github/container/install-pax.sh

```diff
@@ -117,17 +118,41 @@ PAXML_INSTALLED_DIR=${INSTALL_DIR}/paxml
 git clone ${PAXML_REPO} ${PAXML_INSTALLED_DIR}
 pushd ${PAXML_INSTALLED_DIR}
 git checkout ${PAXML_REF}
+if [[ $(uname -m) == "aarch64" ]]; then
+  # These dependencies are broken on ARM64 right now, we handle them separately
+  pip install chex==0.1.7
+  sed -i 's/^tensorflow/#tensorflow/'  paxml/pip_package/requirements.txt requirements.in
+  sed -i 's/^lingvo/#lingvo/' paxml/pip_package/requirements.txt requirements.in
+  sed -i 's/^scikit-learn/#scikit-learn/' paxml/pip_package/requirements.txt requirements.in
+  sed -i 's/^t5/#t5/' paxml/pip_package/requirements.txt requirements.in
+  sed -i 's/^jax/#jax/' paxml/pip_package/requirements.txt requirements.in
+  sed -i 's/^protobuf/#protobuf/' paxml/pip_package/requirements.txt requirements.in
+  sed -i 's/^numpy/#numpy/' paxml/pip_package/requirements.txt requirements.in
+fi
 popd
 
 # SKIP_HEAD_INSTALLS avoids having to install jax from Github source so that
 # we do not overwrite the jax that was already installed. Jax at head is
 # required by both praxis and paxml, and fiddle at head is only required by
 # praxis
 HEAD_PACKAGES="jax fiddle"
-SKIP_HEAD_INSTALLS=true maybe_defer_pip_install -e ${PRAXIS_INSTALLED_DIR} -e ${PAXML_INSTALLED_DIR}[gpu] $HEAD_PACKAGES
+SKIP_HEAD_INSTALLS=true maybe_defer_pip_install ${PRAXIS_INSTALLED_DIR}
+SKIP_HEAD_INSTALLS=true maybe_defer_pip_install ${PAXML_INSTALLED_DIR}[gpu] $HEAD_PACKAGES
+
+if [[ $(uname -m) == "aarch64" ]]; then
+  # array-record pip package is broken on ARM64 right now, we build it from source here.
+  cd ${INSTALL_DIR}
+  pip uninstall -y array-record || true
+  git clone http://github.com/google/array_record.git
+  pushd array_record
+  oss/build_whl.sh
+  pip install  /tmp/array_record/all_dist/array_record-*.whl --force-reinstall
+  popd
+  rm -Rf array_record /tmp/array_record
+fi
```

Comment:
This will likely stay "forever": Google does not intend to provide an arm64 wheel here.

I could pull this into another script if you'd like though!